pkg/resources/grant_privileges_to_share_acceptance_test.go

```diff
@@ -338,6 +337,100 @@ func TestAcc_GrantPrivilegesToShare_OnTag(t *testing.T) {
 	})
 }
 
+func TestAcc_GrantPrivilegesToShare_OnSchemaObject_OnFunction(t *testing.T) {
+	acc.TestAccPreCheck(t)
```

Comment:
in other tests we use `_ = testenvs.GetOrSkipTest(t, testenvs.EnableAcceptance)`, is it not enough here?

Reply:
As discussed: let's add it for now, because it may succeed fortunately by some other test set up.

Reply:
Done.

Reply:
this `acc.TestAccPreCheck(t)` should be now moved to `PreCheck` - you can do it in the next PR